src/components/markdown-help.jsx

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import Markdown from './markdown';
+import simpleAvatar from '../images/simple-avatar.png';
```

Comment:
Let's not import an image like this. Use a hosted service, like https://placeholder.com, for a placeholder as the image example instead.